docs/HyperIndex/terms-of-service.md

```diff
@@ -29,15 +29,27 @@ _The fine print: Please note these terms are intended to protect us, where adver
 
 4. **Payments and Refunds**
 
-   Our products are currently provided on a free-to-use basis. We reserve the right to introduce charges for these services at our discretion. Any pricing changes will be communicated to users in advance.
+   Our services are provided on a paid basis. By subscribing or making a payment, you agree to the applicable fees and billing terms.
+
+   Billing & Payment: Fees are charged upfront based on the selected plan. Any additional unit fees exceeding the included base usage will be billed separately at the end of the calendar month. Payments must be made via the payment methods we support.
```

Comment:
We might decide to start sending an invoice earlier if the extra hours bill exceeds X amount. We need to decide how we want to implement this.

Reply:
I know we spoke about this, will lean over and double check if we're happy with 2.5x